source/features/parse-backticks.tsx

```diff
@@ -20,16 +20,12 @@ function init(): void {
 		'.js-details-container .pr-1 > code', // `isCompare` pushed commits
 		'.Box-row .mb-1 a', // `isCompare` open Pull Request title
 		'.blame-commit-message', // `isBlame` commit message
-		'a[id^="issue_"]', // `isConversationList` issue and PR title
-		'.TimelineItem-body > del', // `isIssue`, `isPRConversation` title edits
-		'.TimelineItem-body > ins', // `isIssue`, `isPRConversation` title edits
 		'[id^=ref-issue-]', // `isIssue` issue and PR references
 		'[id^=ref-pullrequest-]', // `isPRConversation` issue and PR references
```

Comment:
I think more can be dropped here. I think for example these 2 select the commits that appear in conversations (https://github.com/sindresorhus/refined-github/pull/3812) and those are already formatted.

Reply:
I did not finish this. I started doing it and then decided to wait until we drop GHE. There is just to much there and I want to wait. (2 more weeks)